tests/api.rs

```diff
@@ -1,14 +1,7 @@
-// this exists primarily to test various API usages of scroll; e.g., must compile
```

Comment:
i don't think this comment needed to be deleted

Reply:
thx, reverted